yarn.lock

```diff
@@ -64,6 +64,25 @@
   resolved "https://registry.yarnpkg.com/@types/parse-json/-/parse-json-4.0.0.tgz#2f8bb441434d163b35fb8ffdccd7138927ffb8c0"
   integrity sha512-//oorEZjL6sbPcKUaCdIGlIUeH26mgzimjBB77G6XRgnDl/L5wOnpyBGRe/Mmf5CVW3PwEBE1NjiMZ/ssFh4wA==
 
+"@types/prop-types@*":
```

Comment:
Hey @akshayjain04 - Do we need proptypes ? 🤔 Or was it automatically added ?

Reply:
Hey @peak-jiteshahuja, it was automatically added